.github/workflows/actuated-aarch64-test.yaml

```diff
@@ -0,0 +1,66 @@
+name: Actuated aarch64 test
+
+on: [push, pull_request]
+
+# Cancel any preceding run on the pull request.
+concurrency:
+  group: actuated-test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/criu-dev' }}
+
+jobs:
+  build:
+    # Actuated runners are only available are not available in all repositories.
```

Comment:
*Actuated runners are not available in all repositories.

Reply:
changed